oshi-core/src/main/java/oshi/hardware/CentralProcessor.java

```diff
@@ -36,38 +36,38 @@
      *
      * @return vendor string.
      */
-    String getVendor();
+    @Deprecated String getVendor();
 
     /**
      * Name, eg. Intel(R) Core(TM)2 Duo CPU T7300 @ 2.00GHz
      *
      * @return Processor name.
      */
-    String getName();
+    @Deprecated String getName();
```

Comment:
These are valid, but for style purposes add a new line between the `@Deprecated` annotation and the `get`.  You also need to add the (lowercase) `@deprecated` annotation in the Javadoc, pointing to the new location this information will be.  Such as `@deprecated Replaced by {@link ProcessorIdentifier#getName}`